errbot/backends/irc.py

```diff
@@ -299,12 +313,32 @@ def __init__(self,
             self.send_public_message = rate_limited(channel_rate)(self.send_public_message)
         self._reconnect_on_kick = reconnect_on_kick
         self._pending_transfers = {}
+        self.__recently_joined_to = set()
 
         self.nickserv_password = nickserv_password
         if username is None:
             username = nickname
         self.transfers = {}
         super().__init__([(server, port, password)], nickname, username, reconnection_interval=reconnect_on_disconnect)
+        # DEBUG HELPER
+        # Uncomment this to get an "All events" handler for any irc command/message sent
+        #  to the server, don't forget to uncomment __all_events too!.
+        #self.connection.add_global_handler("all_events", self.__all_events, 98)
+
+    # DEBUG HELPER
```

Comment:
I am not fan of commented out code, can you make an if and see if the logger for this module is at DEBUG level and log it then ?